linux_os/guide/system/permissions/restrictions/enable_execshield_settings/sysctl_kernel_exec_shield/bash/shared.sh

```diff
@@ -19,7 +19,6 @@ if [ "$(getconf LONG_BIT)" = "64" ] ; then
   if grep --silent noexec {{{ grub2_boot_path }}}/grub*.cfg ; then
         sed -i "s/noexec.*//g" /etc/default/grub
         sed -i "s/noexec.*//g" /etc/grub.d/*
-        GRUBCFG={{{ grub2_boot_path }}}/*.cfg
-        grub2-mkconfig -o "$GRUBCFG"
+        grub2-mkconfig -o "{{{ grub2_boot_path }}}/grub*.cfg"
```

Comment:
Glob still needs to go outside of the quote, otherwise you end up passing the literal `*` as a file-path character to `grub2-mkconfig` instead of having the shell expand it. :-) Sorry!

Reply:
Aha! Sorry for that. It should be fixed now